setup.py

```diff
@@ -38,9 +38,8 @@
     platforms=['linux'],
     packages=['pysipp'],
     install_requires=['pluggy==0.3.1'],
-    extras_require={
-        'testing': ['pytest'],
-    },
+    setup_requires=['pytest-runner'],
```

Comment:
I like it, but maybe we should adjust to [avoid installation on each `setup.py` call](http://pythonhosted.org/pytest-runner/#considerations)?